RELEASE_NOTES.md

```diff
@@ -174,16 +188,17 @@ Please let us know if we can help!
 We've just released v0.2.0 of shorebird 🎉 :shorebird:
 
 ✨ Highlights:
+
```

Comment:
I ran the markdown formatter on this file to standardize the formatting so the rest of these changes are just formatter changes